FirebaseTestProject/Views/PostRow.swift

```diff
@@ -8,23 +8,95 @@
 import SwiftUI
 
 struct PostRow: View {
-    let post: Post
+    @Binding var post: Post
+    @State var isPresenting: Bool = false
+    @State private var showAlert: Bool = false
+    @State private var systemImage: String = ""
+    let deletePostAction:((Post) -> Void)
+    
+    init(post: Binding<Post>, deletePostAction: @escaping (Post) -> Void) {
```

Comment:
Why is this a binding here?

Reply:
Because we need to modify the recipe then send it back to the original PostsList. Similar to how we were modifying the Ingredient and Direction previously.